erigon-lib/state/inverted_index.go

```diff
@@ -250,15 +250,31 @@ func (ii *InvertedIndex) BuildMissedAccessors(ctx context.Context, g *errgroup.G
 
 }
 
-func (ii *InvertedIndex) openDirtyFiles() error {
+type steps struct {
+	from uint64
+	to   uint64
+}
+
+func (ii *InvertedIndex) openDirtyFiles(fNames []string) error {
 	var invalidFileItems []*filesItem
 	invalidFileItemsLock := sync.Mutex{}
+	stepNameMap := make(map[steps]string, len(fNames))
+	for _, filename := range fNames {
+		from, to, err := ParseStepsFromFileName(filename)
+		if err != nil {
+			continue
+		}
+		stepNameMap[steps{from: from, to: to}] = filename
+	}
 	ii.dirtyFiles.Walk(func(items []*filesItem) bool {
 		for _, item := range items {
 			item := item
 			fromStep, toStep := item.startTxNum/ii.aggregationStep, item.endTxNum/ii.aggregationStep
 			if item.decompressor == nil {
-				fPath := ii.efFilePath(fromStep, toStep)
+				fPath, ok := stepNameMap[steps{from: fromStep, to: toStep}]
```

Comment:
1. this func open not only this files but also: `fPath := ii.efAccessorFilePath(fromStep, toStep)`. And accessor can have different version with data file. 
2. similar func exists in History/Domain